cmd/utils.go

```diff
@@ -130,6 +130,9 @@ func getWriteQuorum(drive int) int {
 	return quorum
 }
 
+// CloneMSS is an exposed function of cloneMSS for gateway usage.
+var CloneMSS = cloneMSS
```

Comment:
Just rename it?

Reply:
Wanted to avoid other changes clobbering the review of the PR. Let me do it in a separate PR